build.gradle

```diff
@@ -20,5 +20,9 @@ test {
 }
 
 application {
-    mainClassName = 'com.connectfour.ConnectFour'
+    getMainClass().set('com.connectfour.ConnectFour')
```

Comment:
What's the advantage of doing this over what you had before?

Reply:
Gradle was giving a notice every time it ran that this was deprecated. Apparently this is the right way to do it now.